modules/workflows/OrcaRecoveryWorkflow/orca_recover_workflow.asl.json

```diff
@@ -14,6 +13,7 @@
       },
       "Type": "Task",
       "Resource": "${orca_lambda_extract_filepaths_for_granule_arn}",
+      "ResultPath": "$.output_extract_filepath_from_granules",
```

Comment:
Could say `"ResultPath": "$.input"` here, and remove the `"input.$": "$.output_extract_filepath_from_granules"` line below, unless I'm missing something.

Reply:
getting 

```
{
  "error": "KeyError",
  "cause": {
    "errorMessage": "'input'",
    "errorType": "KeyError",
    "requestId": "17b045c2-66cb-4972-acc0-07f7fd2ef29c",
    "stackTrace": [
      "  File \"/var/task/aws_lambda_powertools/logging/logger.py\", line 358, in decorate\n    return lambda_handler(event, context, *args, **kwargs)\n",
      "  File \"/var/task/request_from_archive.py\", line 667, in handler\n    _VALIDATE_INPUT(event[\"input\"])\n"
    ]
  }
}
```

Reply:
Ah, I suppose it would place it under `event["event"]["input"]` which isn't great.
Let's just generalize to something like `"ResultPath": "$.previous_step_output"` and call it good.

Reply:
updated